src/Symfony/Component/Translation/Bridge/Loco/LocoProvider.php

```diff
@@ -117,7 +118,18 @@ public function read(array $domains, array $locales): TranslatorBag
                 throw new ProviderException('Unable to read the Loco response: '.$responseContent, $response);
             }
 
-            $translatorBag->addCatalogue($this->loader->load($responseContent, $locale, $domain));
+            $locoCatalogue = $this->loader->load($responseContent, $locale, $domain);
+            $catalogue = new MessageCatalogue($locale);
+
+            foreach ($locoCatalogue->all($domain) as $key => $message) {
+                if (str_starts_with($key, $domain.'__')) {
```

Comment:
What if the key starts with another domain ?

Reply:
It can't happen, as we fetch catalogue by catalogue, filtered on domain https://github.com/symfony/symfony/pull/43990/files#diff-b3090d6706b3a0d9e2e7674de3c1029810357aa3eb493250c9812cafa44f8232L94